### PR TITLE
Add consistency to STDIN-getopt arguments

### PR DESCRIPTION
**Pull request by marxsk**
* Longopt options can be used on stdin either with dash or underscores
* Tests were not updated yet as there might be changes in XML metadata

### CODE REVIEW

**Comment by marxsk** [comment]
@oalbrigt please, take a look at code

File: fence/agents/vbox/fence_vbox.py [hunk 1]
```diff
@@ -21,66 +22,82 @@
...
-    return options.get("--uuid") or options.get("--plug")
-
-_domain_re = re.compile(r'^\"(.*)\" \{(.*)\}$')
-
-
 def _invoke(conn, options, *cmd):
     prefix = options["--sudo-path"] + " " if "--use-sudo" in options else ""
-    conn.sendline(prefix + "VBoxManage " + " ".join(cmd))
+    conn.send_eol(prefix + options["--vboxmanage-path"] + " " + " ".join(cmd))
+    conn.log_expect(options["--command-prompt"], int(options["--shell-timeout"]))
```
Comment (oalbrigt) [inline fence/agents/vbox/fence_vbox.py]: Should it check for it do the same log_expect twice?

Comment (marxsk) [inline fence/agents/vbox/fence_vbox.py]: Aah, I just wanted to reuse branch. fence_vbox should not be part of this commit.

**Comment by marxsk** [comment]
Applied in https://github.com/ClusterLabs/fence-agents/commit/de490e0590dabc8b30619092f9197ab0d4c47458
